src/reducers/playersListReducer.js

```diff
@@ -1,6 +1,6 @@
 export default (state = [], action) => {
   switch (action.type) {
-    case 'GET_TEAMS':
+    case 'SET_PLAYERS':
```

Comment:
Let's also add another reducer case above this called `RESET_PLAYERS` and have it look like so:

```
const initialPlayers = new Array(20).fill({});

export default (state = initialPlayers, action) => {
  switch (action.type) {
    case 'RESET_PLAYERS':
      return initialPlayers;
    case 'SET_PLAYERS':
      return action.payload;
    default:
      return state;
  }
};
```

And then in our `actions.js` file, wherever you see:

`dispatch({ type: 'SET_PLAYERS', payload: resetPlayers });`

replace it with:

`dispatch({ type: 'RESET_PLAYERS' });`

and remove the `resetPlayers` variable. The reasoning being that our reducer should always handle the reset/initialization in it's own edge case.